### PR TITLE
Enhancement: Add a PR template to guide contributors in what to put in their PRs.

### PR DESCRIPTION
**Pull request by wldcordeiro**
Associated Issue: N/A

## Summary of Changes

* add a PR template

@jlongster @jasonLaster this a very bare bones template but I figured it's a good start. I'll adjust it to any feedback.

### CODE REVIEW

**Comment by jlongster** [comment]
@clarkbw Probably good to have your feedback here. I'm not sure I feel the current template is very helpful, as there may not be an issue to associate with or necessary to have a screenshot/video. But you can decide what is good.

**Comment by wldcordeiro** [comment]
@jlongster I lifted this from the template I use at work. I'll change it to what makes sense but figured it's a good conversation starting point. 

> I'm not sure I feel the current template is very helpful, as there may not be an issue to associate with or necessary to have a screenshot/video.

Usually if there isn't an associated issue we change the line to `Associated Issue: N/A` and if there aren't screenshots/videos we just drop the sections out and give a brief summary in the change list.

File: .github/PULL_REQUEST_TEMPLATE [hunk 1]
```diff
@@ -0,0 +1,10 @@
+Associated Issue: #<issue number>
+
+## Summary of Changes
+
+* change 1
+* change 2
+
+## Screenshots (OPTIONAL)
+
+## Video (OPTIONAL)
```
Comment (jasonLaster) [inline .github/PULL_REQUEST_TEMPLATE]: Hmm, maybe combine Screenshots and Video.

File: .github/PULL_REQUEST_TEMPLATE [hunk 2]
```diff
@@ -0,0 +1,10 @@
+Associated Issue: #<issue number>
+
+## Summary of Changes
```
Comment (jasonLaster) [inline .github/PULL_REQUEST_TEMPLATE]: Maybe a lesser header `####` or just bold `**`

**Comment by clarkbw** [comment]
I think this looks like a good start, lets try it out and update it as we go.

**Pull request merged**